extensions/geode-modules-tomcat8/src/main/java/org/apache/geode/modules/session/catalina/DeltaSession8.java

```diff
@@ -23,6 +23,7 @@
    * Construct a new <code>Session</code> associated with no <code>Manager</code>. The
    * <code>Manager</code> will be assigned later using {@link #setOwner(Object)}.
    */
+  @SuppressWarnings("unused")
```

Comment:
Thanks for adding these - I think they will help if we ever need to do another round of cleanup.